pkg/rancher/rancher.go

```diff
@@ -261,6 +262,10 @@ func (r *Rancher) Start(ctx context.Context) error {
 		return err
 	}
 
+	if features.MCM.Enabled() {
```

Comment:
Changed registration from `dashboardapi` to top level, as the nodedriver handler does not need to run in the clsuter agent (and was breaking CI).